src/tir/schedule/analysis/analysis.cc

```diff
@@ -298,5 +298,35 @@ Array<StmtSRef> GetChildBlocks(const ScheduleState& self, const StmtSRef& parent
   throw;
 }
 
+Array<Stmt> GetChildren(const Stmt& stmt) {
+  /*! \note Nested SeqStmt is not allowed in schedule. */
```

Comment:
I think it is TIR, not a schedule? (Line 314 also has such expression.)
```suggestion
  /*! \note Nested SeqStmt is not allowed in TIR. */
```